### PR TITLE
Use prop types from `deprecated-react-native-prop-types`

### PR DESCRIPTION
**Pull request by renchap**
Starting with React Native 0.68, using Prop Types from `react-native` emits a warning because this is deprecated:

> ViewPropTypes will be removed from React Native. Migrate to ViewPropTypes exported from 'deprecated-react-native-prop-types'.

This PR simply loads them from the new `deprecated-react-native-prop-types` package, as recommended in the warning above.

### CODE REVIEW

**Comment by apfritts** [comment]
Tested in our own app... I second this!

**Comment by gkpo** [comment]
Tested here too and it fixes the warning. Would be great to merge it

**Comment by lockieluke** [comment]
PLEASE MERGE IT THIS

**Comment by mwalkerr** [comment]
Also would like to see this merged

**Review by freeboub — COMMENTED** [review]
@renchap can you pleas fix the warning

**Comment by hontas** [comment]
@renchap there is a lint warning in `Video.js`: `Strings must use singlequote`.
If that's what keeping this PR from being merged, can you please update it?

File: Video.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import { StyleSheet, requireNativeComponent, NativeModules, UIManager, View, ViewPropTypes, Image, Platform, findNodeHandle } from 'react-native';
+import { StyleSheet, requireNativeComponent, NativeModules, UIManager, View, Image, Platform, findNodeHandle } from 'react-native';
+import { ViewPropTypes } from "deprecated-react-native-prop-types";
```
Comment (elliottkember) [inline Video.js]:
```suggestion
import { ViewPropTypes } from 'deprecated-react-native-prop-types';
```

**Review by freeboub — APPROVED** [review]

**Pull request merged**